larg4/Core/larg4Main_module.cc

```diff
@@ -124,8 +126,9 @@ larg4::larg4Main::larg4Main(fhicl::ParameterSet const & p)
   session_(0),
   UI_(0),
   seed_(p.get<long>("seed", -1)),
-  macroPath_( p.get<std::string>("macroPath",".")),
+  macroPath_( p.get<std::string>("macroPath","FW_SEARCH_PATH")),
   pathFinder_( macroPath_),
+  g4MacroFile_( p.get<std::string>("visMacro", "larg4.mac")),
```

Comment:
IIRC, the artists wanted users to try and avoid default values in the FCL parameter lookups as it helps to catch typos in the FCL file parameters.  Would the workflow be ruined if the defaults were taken out?

Reply:
@absolution1, we strongly discourage defaults for *physics* parameters.  As these parameters have to do with G4 macros, defaults are probably fine.

Reply:
No, it should be fine without the defaults, but I think it would be beneficial to check the FW_SEARCH_PATH. This is what is done in Legacy.